controllers/UsersController.js

```diff
@@ -0,0 +1,15 @@
+const {fetchUsers} = require('../models/usersModel')
+
+
+
+
+const getUsers = (req, res) => {
+    fetchUsers()
+        .then((topics) => {
+            res.status(200).send(topics.rows);
```

Comment:
send an object with a key of topics - clearer in your test so you can access body.topics not just body. Will help you in front end too when you use your server as an API